src/webdriver_kaifuku/__init__.py

```diff
@@ -31,7 +32,15 @@ def _get_browser_name(webdriver_kwargs: dict) -> str:
     """
     Extract the name of the browser from the desired capabilities
     """
-    name = webdriver_kwargs.get("desired_capabilities", {}).get("browserName")
+    # Options is an Object with "to_capabilities" as a method
+
+    name_from_options = (
+        webdriver_kwargs.get("options", ArgOptions()).to_capabilities().get("browserName")
```

Comment:
the options value might not be a ArgOptions intance, how to reconcile that?

Reply:
I think it this case it really does not matter. I was just looking for a class that extends BaseOptions, which is an abstract class and thus it cannot be instantiated. I used it as a mean to have a class with a `to_capabilities()` method that can be used.
However, maybe it is too obscure and we should be more explicit:
```
name_from_options = webdriver_kwargs.get("options").to_capabilities().get("browserName") if "options" in webdriver_kwargs else None
```
Would the second approach be more clear?

Reply:
What i meant was that the input may be json/yaml data and we might need to cast to the options somehow

What would the input look like if it came from insights qe or sattelite qe config files?

Reply:
In my opinion, that is a new feature outside the scope of this PR (although we can combine). If my understanding is correct, [BrowserManager](https://github.com/RonnyPfannschmidt/webdriver_kaifuku/blob/main/src/webdriver_kaifuku/__init__.py#L117) just passes the options as they come to the [Selenium WebDriver](https://github.com/RonnyPfannschmidt/webdriver_kaifuku/blob/main/src/webdriver_kaifuku/__init__.py#L55). Selenium expects an [Options or a list of Options](https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/remote/webdriver.py#L203). 
The current implementation cannot handle options in another format (json/yaml), that's why I think that is a new feature request. However, it can already handle `Options` created in Python code, but it will fail when doing so as it won't be able to parse the browser name correctly.

So, regarding this PR, do you think we should expand the scope to also add the option to cast options from json/yaml into `Options`? Or should we address that separately?

The is an error in this PR, as WebDriver allows a single `Options` object or a list of them. The current implementation in this PR cannot handle a list. Do you think it is a fair approach to call the [create_matches](https://github.com/SeleniumHQ/selenium/blob/036ee7f6aec797f5fe4608d2f7673e59d7f548b5/py/selenium/webdriver/remote/webdriver.py#L141) function as Selenium [does] or is it tightening it too much to the current Selenium implementation? Any ideas?

Reply:
It's a bit unclear to me, I'll have to take some time to understand the context